lib/src/states/camera_picker_state.dart

```diff
@@ -1635,6 +1635,14 @@ class CameraPickerState extends State<CameraPicker>
     );
   }
 
+  TextDirection determineTextDirection(Enum orientation, DominantHand hand) {
```

Comment:
Use `DeviceOrientation` instead of `Enum`?

Reply:
That was my initial thought too, but the code on line 1652 cause issues, specifically:
`final orientation = deviceOrientation ?? MediaQuery.of(context).orientation`
it's a bit tricky. `deviceOrientation` is enum of type `DeviceOrientation`, but `MediaQuery.of(context).orientation` is enum of type `Orientation`. The type for final orientation is therefore `Enum`. `DeviceOrientation` doesn't inherit from `Orientation`.

Reply:
1652 (if MediaQuery-value is set) will crash 1678 (`verticalDirection: orientation == DeviceOrientation.portraitDown`)
I'll post suggestion for work-around.

Reply:
Turn those values into string so you don't need an `Enum`, like what we've already did.